zds/forum/views.py

```diff
@@ -262,6 +262,8 @@ def dispatch(self, request, *args, **kwargs):
         if ('text' in request.POST or request.method == 'GET') \
                 and self.object.author != request.user and not request.user.has_perm('forum.change_topic'):
             raise PermissionDenied
+        if self.object.first_post().is_visible == False and not request.user_has_perm('forum.change_topic'):
```

Comment:
if self.object.first_post().is_visible == False => if not self.object.first_post().is_visible